packages-ts/gauntlet-terra-contracts/src/commands/tooling/updateContractAdmin.ts

```diff
@@ -0,0 +1,80 @@
+import { Result } from '@chainlink/gauntlet-core'
+import { logger, prompt } from '@chainlink/gauntlet-core/dist/utils'
+import { TerraCommand, TransactionResponse } from '@chainlink/gauntlet-terra'
+import { AccAddress, MsgUpdateContractAdmin } from '@terra-money/terra.js'
+import { CATEGORIES } from '../../lib/constants'
+
+type CommandInput = {
+  newAdmin: string
+  contract: string
+  forceNewAdmin?: boolean
```

Comment:
I'd suggest `--force` or `--unsafe` or `--skip-check-multisig`.

For example we could have a global `--unsafe` aliased as `--skip-checks`. This one would skip all checks. Then if we label specific checks, we could have auto flags like `--skip-check-${checkName}` to skip specific checks in specific situations (e.g., testing).